src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -460,7 +460,7 @@ static u8string OpenSystemFileBrowser(bool isSave)
             if (buffer.empty())
             {
                 // Use localised "Unnamed Park" if park name was empty.
-                buffer = FormatStringID(STR_UNNAMED_PARK, nullptr);
+                buffer = FormatStringIDLegacy(STR_UNNAMED_PARK, nullptr);
```

Comment:
Anything that is calling this with a nullptr is probably incorrect it should probably be calling `LanguageGetString`

Reply:
I’m already on this.